simba/costs.py

```diff
@@ -101,20 +101,42 @@ def calculate_costs(c_params, scenario, schedule, args):
 
     # CHARGING INFRASTRUCTURE
     cs = schedule.scenario["components"]["charging_stations"]
-    # depot charging stations - each charging bus generates one CS
-    for csID in cs.values():
-        if csID["type"] == "deps":
-            costs["c_cs"] += c_params["cs"]["capex_deps_per_kW"] * csID["max_power"]
-    # opportunity charging stations - nr of CS depend on max nr of simultaneously occupied CS
     for gcID in gcs:
-        if schedule.stations[gcID]["type"] == "opps":
+        # depot charging stations - each charging bus generates one CS
+        if schedule.stations[gcID]["type"] == "deps":
```

Comment:
This assumes that station names are always identical to GCs. Why not just iterate over schedule.stations?
```
for gcID, station in schedule.stations.items():
  if station["type"] == "deps":
    ...
```
This makes the following code much more readable.